app/views/content/blog/application-tips-from-a-teacher-training-provider.md

```diff
@@ -57,9 +57,13 @@ I can’t emphasise enough how important it is to avoid spelling mistakes. Make
 
 ## References
 
-And finally, we turn to references. Who you pick is really important. You will need two references. One could be a university or academic tutor and the second could be from a school or somewhere you have worked in a voluntary capacity, or it could be someone from your current employment.
+And finally, we turn to references. You'll need 2 referees.
```

Comment:
Just a query about whether/how we should edit a blog written by someone else, even for factual updates? I think it would be worth contacting the Content team to see if they have any guidance on this (eg do we need to make it clear that we've updated it in a footer note or something?)

Reply:
shall speak to the content team